exercises/concept/intro-select/report-results.sh

```diff
@@ -0,0 +1,15 @@
+DB_FILE=$1
```

Comment:
Add the shebang even if you specifically invoke it with the interpreter.
```suggestion
#!/usr/bin/env bash
DB_FILE=$1
```